src/pairwise.jl

```diff
@@ -178,7 +189,7 @@ function _pairwise(::Val{skipmissing}, f, x, y, symmetric::Bool) where {skipmiss
         # but using `promote_type` rather than `promote_typejoin`)
         U = mapreduce(typeof, promote_type, dest)
         # V is inferred (contrary to U), but it only gives an upper bound for U
-        V = promote_type_union(Union{T, Tsm})
+        V = promote_type_union(Union{T,Tsm})
```

Comment:
```suggestion
        V = promote_type_union(Union{T, Tsm})
```

Reply:
For whitespace I'm largely relying on the Julia extension for VSCode with the command F1 -> Format Document, and that gives no space after the comma in this example. Is there an alternative approach to automatic code formatting that's preferred for StatsBase?